src/pynwb/misc.py

```diff
@@ -267,15 +268,20 @@ class DecompositionSeries(TimeSeries):
              'doc': 'a table for describing the frequency bands that the signal was decomposed into', 'default': None},
             {'name': 'source_timeseries', 'type': TimeSeries,
              'doc': 'the input TimeSeries from this analysis', 'default': None},
+            {'name': 'source_channels', 'type': DynamicTableRegion, 'doc': 'the channels that provided the source data',
```

Comment:
It would be useful to indicate in the doc string for ``source_channels`` where this DynamicTableRegion points to. E.g., " In the case of electrical recordings this is typically a DynamicTableRegion pointing to the ElectrodesTable as part of NWBFile.electrodes, similar to ElectricalSeries.electrodes." If this is different for an ImageSeries than it would be nice to mention this as well.